tensorflow/workspace.bzl

```diff
@@ -212,7 +212,7 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
   )
 
   native.http_archive(
-      name = "protobuf",
+      name = "protobuf_archive",
```

Comment:
"//external:protobuf" is [referenced here](https://github.com/llhe/tensorflow/blob/19bf520f381b50086aadb2eb41eabfeca55d0e3e/third_party/grpc.BUILD#L1276).